### ISSUE TITLE
Change the python scatter

### ISSUE THREAD

**Issue by RedGl0w** [Omega-Numworks/Omega#328]
#### What I want to see in the next version of Omega
I would like to see in the python editor the scatter stops at the last line, like on every editors :trollface: 
![image](https://user-images.githubusercontent.com/43498612/79040041-01233300-7be6-11ea-9c0e-d5ecefeee822.png)

**Comment by windows9x95** [comment]
I agree with the idea, if this isn't hard to do, I can try to do it :)

**Comment by RedGl0w** [comment]
> I agree with the idea, if this isn't hard to do, I can try to do it :)

I don't think it's too hard, but that's not the easiest thing
